lib/src/Base/Common/ResourceMap.cxx

```diff
@@ -737,6 +737,9 @@ void ResourceMap::loadDefaultConfiguration()
   addAsScalar("AbdoRackwitz-DefaultTau", 0.5);
   addAsUnsignedInteger("AbdoRackwitz-DefaultMaximumEvaluationNumber", 100000);
 
+  // MultiStart parameters //
+  addAsBool("MultiStart-KeepResults", true);
```

Comment:
why not having in mind a `storeResult` with `UnsignedInteger` ? We might set `0` if we don't want to store any result or limit the sizing (for example, default value = `1000`)

Reply:
I don't see the point of keeping only a part of the results. The use-case behind the storage of the results was to partition the starting points according to their corresponding optimal point, and when it is not needed I propose to deactivate it. With your proposal, it becomes less straightforward to keep everything: you have to set the bound to the size of the starting points sample, which cannot be set in the ResourceMap. Do you have a use-case where only a part of these results would be meaningful?

Reply:
no use case for the moment